src/pokemon_storage_system.c

```diff
@@ -1218,7 +1218,7 @@ static const u16 gWallpaperPalettes_Plain[][16] =
 static const u32 gWallpaperTiles_Plain[] = INCBIN_U32("graphics/pokemon_storage/plain.4bpp.lz");
 static const u32 gWallpaperTilemap_Plain[] = INCBIN_U32("graphics/pokemon_storage/plain.bin.lz");
 
-// 12�18 tilemap
+// 12�18 tilemap
```

Comment:
Broken encoding?

Reply:
Yeah, I'm not sure what that is?

Reply:
Improperly configured editor perhaps? Just replace the broken Unicode character.

Reply:
The x was a unicode character, for whatever reason. I had the same problem in another PR. Just replace it with a normal ascii `x`